x-pack/test/performance/journeys/security_detections_dashboard/config.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { FtrConfigProviderContext } from '@kbn/test';
+import { serializeApmGlobalLabels } from '../../utils';
+
+export default async function ({ readConfigFile }: FtrConfigProviderContext) {
+  const performanceConfig = await readConfigFile(require.resolve('../base.config'));
+
+  const testFiles = [require.resolve('./security_detections_dashboard')];
+
+  const config = {
+    testFiles,
+    ...performanceConfig.getAll(),
+    scalabilitySetup: {
+      warmup: [
+        {
+          action: 'constantConcurrentUsers',
+          maxUsersCount: 10,
+          duration: '30s',
+        },
+        {
+          action: 'rampConcurrentUsers',
+          minUsersCount: 10,
+          maxUsersCount: 50,
+          duration: '2m',
+        },
+      ],
+      test: [
+        {
+          action: 'constantConcurrentUsers',
+          maxUsersCount: 50,
+          duration: '5m',
+        },
+      ],
+      maxDuration: '10m',
+    },
```

Comment:
I don't think you need it. `scalabilitySetup` is opt-in option to get Kibana server requests from APM, in case you wanna run Kibana server scalability testing. IF your goal is ES Rally input, you can safely remove it

Reply:
@dmlemeshko The initial goal is ES rally input but in in the future we will probably want to include some security journeys in the Kibana server scalability testing.  If we're limiting or trying to curate the journeys included in the scalability testing I have no objection to removing this and reconsidering which security journeys to add/create in the future.

Reply:
Yes, I would say we are trying to limit number of scalability journeys at the moment because it is in a "testing" phase.
I suggest we remove it for now and as soon as scalability pipeline is fully ready, I will send an update and we can discuss which journeys Security Team would like to add for run.